src-java/kilda-model/src/main/java/org/openkilda/model/Isl.java

```diff
@@ -116,9 +117,17 @@
     @Property(name = "under_maintenance")
     private boolean underMaintenance;
 
+    // TODO(surabujin): drop after migration
     @Property(name = "enable_bfd")
     private boolean enableBfd;
 
+    @Property("bfd_interval")
+    @Convert(graphPropertyType = Long.class)
+    private Duration bfdInterval;
+
+    @Property("bfd_multiplier")
+    private short bfdMultiplier;
```

Comment:
should we add new fields into `toString()` method? or just remove `toString()` to generate it by `@Data`?

Reply:
Can't say for sure.

Looks like `.toString()` include a brief and only most important view of `Isl` object. Added fields do not part of the most important fields of `Isl` object, from my point of view. I would like not to include them in the `.toString()` view now.